proof/refine/ARM/CSpace1_R.thy

```diff
@@ -3179,43 +3213,48 @@ lemma n_dest:
 
 end
 
+context begin interpretation Arch .
```

Comment:
Would appreciate FIXME arch-split on these as per arch-split.md
You're also getting a time penalty when you do these. It's much faster to do:
`context Arch begin arch_global naming`
and then `arch_requalify_fact ... (* FIXME arch-split *)`